roles/sriov_config/tasks/create_node_policies.yml

```diff
@@ -6,7 +6,28 @@
   loop_control:
     loop_var: sriov
     label: "{{ sriov.resource }}"
-  when: sriov.node_policy is defined
+  when:
+    - sriov.node_policy is defined
+    - '"OCPBUGS-32139" not in dci_workarounds|default([])'
```

Comment:
IMO, I'm ok with this, but I was wondering if simply we add retries in this task and remove the conditions and new task with the dci_workarounds. Let's hear other opinions.

Reply:
yeah, I would prefer to avoid using `dci_`  vars in the roles. May need to create a new playbook to look through the resources and wait for it it be created.

Reply:
Anyway, retries are the secret sauce of most of the issues : )

Reply:
I know Ramon want's to do things right, but I do not want to image the number of work-arounds we would have in our code if we'll add them for every retry we have added in our plays :see_no_evil:

Reply:
I agree with both, @ramperher could you remove the dci_workarounds in here? It won't hurt adding some retries into the creation of the resource.  It will simplify the task and we won't need to remove it later, win-win. Also I have strong feelings about making these roles dependent of DCI unless necessary.

Reply:
It's updated now, could you take a look again? @manurodriguez @betoredhat @tonyskapunk

Reply:
@ramperher, thanks a lot, for me a work-around it's worth adding when it involves extra tasks (restart a pod, apply a config, etc) but if it's just a retry we could afford not to add it, an in favour of your argument, there is a BUG reported, so that does not happens all the time, so it makes sense to reference it in the code, but let's discuss this in a a grooming or daily meeting to have a touch point, it's been a while we do not talk about work-arounds : )

Reply:
sure, let's do that :) closing this, thanks!